display_blocks.js

```diff
@@ -160,24 +155,106 @@ function parseObject(obj) {
     var nodeInstPVs = document.getElementById("inst_pvs");
     var nodeInstPVList = document.createElement("ul");
 
-    nodeInstPVs.appendChild(nodeInstPVList);
     getDisplayRunInfo(nodeInstPVs, instrumentState.inst_pvs);
+
+    nodeInstTitle.appendChild(document.createTextNode(instrument));
+    nodeConfigTitle.appendChild(document.createTextNode("Configuration: " + instrumentState.config_name));
+
+    document.getElementById("config_name").appendChild(nodeConfigTitle);
 	
 	setVisibilityMode('block');
 }
 
+
+function clearBox(elementID){
+    document.getElementById(elementID).innerHTML = "";
+}
+
+/**
+ * creates a Title at the top looking similar to the IBEX GUI
+ */
+function createTitle(inst_details){
+	clearBox("top_bar");
+	
+	document.getElementById("top_bar").innerHTML = "<div id = \"inst_name\"></div><table><tr id = table_part><th id = \"next_part\" style = \"padding: 10px; background-color:lightgrey ; border: black 2px solid\";></th></tr></table>";
```

Comment:
addition of a style tag to make the width responsive to the page as the only the status of the top bar stretches across the page